cedar-policy-validator/src/schema_file_format.rs

```diff
@@ -275,6 +275,107 @@ impl SchemaType {
     }
 }
 
+impl SchemaTypeVariant {
+    pub fn long_static_top() -> SchemaTypeVariant {
+        SchemaTypeVariant::Long(SchemaLongDetails {
+            bounds_opt: Some(SchemaLongBounds {
+                min: i64::MIN,
+                max: i64::MAX,
+            }),
+        })
+    }
+}
+
+// In the schema file, we want the bounds of a Long attribute to be given as
+// "min" and "max" fields at the same level as `"type": "Long"` with both or
```

Comment:
I'm not convinced they need to be at the same level. I think the code would be cleaner and the interface would be fine if the schema were written as something like:
```
... "foo": {
  "type": "Long",
  "bounds": {
    "min": 0,
    "max": 8
  }
}

Reply:
We should follow what JSON Schema does: https://json-schema.org/understanding-json-schema/reference/numeric.html#range